package.json

```diff
@@ -51,8 +52,8 @@
     "waait": "1.0.5"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "REACT_APP_GIT_SHA=`git rev-parse HEAD` react-scripts build",
+    "start": "react-scripts --max_old_space_size=8196 start",
+    "build": "REACT_APP_GIT_SHA=`git rev-parse HEAD` react-scripts --max_old_space_size=8196 build",
```

Comment:
Do you guys know why we are running into this in the first place. Im curious what is causing the build process to OOM in the first place, that we need to increase this limit

Reply:
It's probably the chart library that we use. We faced a similar issue in our repo.
https://github.com/plotly/react-plotly.js/issues/135

Reply:
@khelif96 I went ahead and used the smallest partial bundle provided by that plotting library we could instead of what we had before. We still need a higher memory limit during building but it reduced the final bundle size, in our repo at least, by a lot.

Reply:
Is there anything you can do to make the plotly library smaller. This plugin including the plotly library is larger then spruce, and all our deps combined. It causes spruces bundle to be twice as large. 
![image](https://user-images.githubusercontent.com/4605522/96657024-17d1e180-130f-11eb-8dad-af972b55bdeb.png)

vs 
![image](https://user-images.githubusercontent.com/4605522/96657090-464fbc80-130f-11eb-9701-0c9ec62ea4ba.png)

You might have already done this but if not build a custom version of plotly using only the modules you need. https://github.com/plotly/plotly.js/#modules

Reply:
Btw this chart can be viewed by looking at the files tab on the compile task

Reply:
Building a custom version of plotly sounds time consuming. It is known that graph charts tend to be huge. If their module is code split from the rest of the app then this should not be a problem because the performance on their page should be the concern of their team. also - if you are requesting that they build a custom version then benchmarks for the loading times with and without the plotly library would be helpful if not necessary. For example, if the load time with plotly lib is under a certain acceptable threshold, e.g. 2000ms, then there is no need to waste time over optimizing.